boards/arm/arty/arty_a7_arm_designstart_m1.dts

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
I'd rather see this definition under dts/arm/_vendor_ , at least the part that defines the SoC (e.g. not the LEDs). That's the convention at least, and it is followed in most Cortex-m SoCs.

Reply:
`arm` is both an architecture and a vendor here, similar to the other SoCs provided by ARM Limited. The soft-core SoC is provided by ARM, but it can be implemented on silicon from various vendors. ARM Limited supports implementing it on at least Xilinx and GOWIN semiconductors FPGAs for now.